CHANGELOG.rst

```diff
@@ -15,19 +23,34 @@ User-Facing Changes
 **🐞 Bug Fixes**
 
 - **UI**: Fix styles for the edit launcher environment list (`#3360 <https://github.com/SwissDataScienceCenter/renku-ui/pull/3360>`__).
+- **UI**: Allow opening a project from Renku 2.0 search if the namespace is missing in the result (`#3353 <https://github.com/SwissDataScienceCenter/renku-ui/pull/3353>`__).
 
 Internal Changes
 ~~~~~~~~~~~~~~~~
 
+**New Features**
+
+- **Data services**: Add support for data connectors (`#407 <https://github.com/SwissDataScienceCenter/renku-data-services/pull/407>`__).
+- **Data services**: Do not synchronize blocked users which from Keycloak (`#393 <https://github.com/SwissDataScienceCenter/renku-data-services/pull/393>`__).
```

Comment:
this sentence doesn't make sense. I guess the `which` shouldn't be there

Reply:
```suggestion
- **Data services**: Do not synchronize blocked users from Keycloak (`#393 <https://github.com/SwissDataScienceCenter/renku-data-services/pull/393>`__).
```